src/main/resources/db/migration/V6__insert_data.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO capability (capabilityName) VALUES
+    ("Software Development"),
+    ("Data Analysis"),
+    ("Cybersecurity Management"),
+    ("Cloud Infrastructure"),
+    ("DevOps Automation")
+;
+
+INSERT INTO band (bandName) VALUES
+    ("Trainee"),
+    ("Associate"),
+    ("Senior"),
+    ("Lead"),
+    ("Principal")
```

Comment:
senior -> senior associate
lead -> consultant
missing -> manager
and over principal: lead :)

Reply:
i apprentice przez trainee